src/guide/best-practices/accessibility.md

```diff
@@ -478,7 +478,7 @@ The World Wide Web Consortium (W3C) Web Accessibility Initiative (WAI) develops
 - [Robust](https://www.w3.org/TR/WCAG21/#robust)
   - Users must be able to access the content as technologies advance
 
-#### Web Accessibility Initiative – Accessible Rich Internet Applications (WAI-ARIA) {#web-accessibility-initiative-–-accessible-rich-internet-applications-wai-aria}
+#### Web Accessibility Initiative – Accessible Rich Internet Applications (WAI-ARIA) {#web-accessibility-initiative-accessible-rich-internet-applications-wai-aria}
```

Comment:
I believe this would be a change to the original anchor that was generated by VitePress. What's the motivation behind changing it?

Reply:
It looks strange (3 dashes), I'm not sure if it's the original anchor that was generated by VitePress or a mistake of my script from #2049. If you think this is okay, feel free to close this one.

Reply:
Btw I found this when sycn these anchor to zh-cn doc. And I found that zh-cn doc is using the one-dash version anchor. Thats why I think it might be a mistake of my script and created this PR.

Reply:
The header with 3 dashes is the same as we had previously. You can see it in the preview deployments for some of the older PRs. I've also checked it locally with an older branch.

Perhaps this dash should be stripped out like other punctuation, but in that case I think it should be changed in VitePress too.

Reply:
okay, feel free to close this if no need to change. thanks.